src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -157,13 +158,20 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
 
   const auto* thread_state_slice =
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
+  auto begin_ns = std::chrono::nanoseconds(thread_state_slice->begin_timestamp_ns());
+  auto end_ns = std::chrono::nanoseconds(thread_state_slice->end_timestamp_ns());
+
   return absl::StrFormat(
       "<b>%s</b><br/>"
       "<i>Thread state</i><br/>"
       "<br/>"
-      "%s",
+      "<br/>"
+      "%s"
+      "<br/>"
+      "<b>Time:</b> %s",
       GetThreadStateName(thread_state_slice->thread_state()),
-      GetThreadStateDescription(thread_state_slice->thread_state()));
+      GetThreadStateDescription(thread_state_slice->thread_state()),
+      orbit_display_formats::GetDisplayTime(absl::FromChrono(end_ns - begin_ns)));
```

Comment:
I think you want to use `TicksToDuration` instead of `absl::FromChrono` to be consistent with the other usages.

Reply:
Done.